Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/Views.kt

```diff
@@ -54,3 +54,27 @@ fun TextView.setUrl(@StringRes textRes: Int, label: String, url: String) {
         }
     }
 }
+
+fun TextView.setUrl(@StringRes textRes: Int, @StringRes labelRes: Int, @StringRes urlRes: Int) {
+    val url = context.getString(urlRes)
+    val label = context.getString(labelRes)
+    context.getString(textRes).also {
+        val indexOf = it.indexOf(label)
+        if (indexOf > 0) {
+            setText(
+                SpannableString(it).apply {
```

Comment:
Maybe Core-ktx `buildSpannedString` can be used here?  I think it will make it nicer